pkg/artifactory/repositories.go

```diff
@@ -30,6 +30,7 @@ type LocalRepositoryBaseParams struct {
 	PropertySets           []string `hcl:"property_sets" json:"propertySets,omitempty"`
 	ArchiveBrowsingEnabled *bool    `hcl:"archive_browsing_enabled" json:"archiveBrowsingEnabled,omitempty"`
 	DownloadRedirect       *bool    `hcl:"download_direct" json:"downloadRedirect,omitempty"`
+	PriorityResolution     *bool    `hcl:"priority_resolution" json:"priorityResolution,omitempty"`
```

Comment:
Please remove the `*` and the `omitempty` we've had never ending problems. And, particularly since this is a `bool` the only possible values are `true` or `false`. `omitempty` will simply leave it up to the server to choose which will confuse everyone

Reply:
done